aten/src/ATen/ConjugateFallback.cpp

```diff
@@ -0,0 +1,122 @@
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <torch/library.h>
+
+#include <ATen/native/UnaryOps.h>
+#include <ATen/NativeFunctions.h>
+
+namespace at {
+
+void conjugateFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
+  // Situations to handle:
+  //  1. Purely functional situation.  Easy: materialize all inputs and
+  //     call it a day.
+  //  2. Inplace operation.  Desugar x.add_(2) into x.conj_().add_(2).conj_().
```

Comment:
why do we have the extra `conj_()` at the end? shouldn't it just be `Desugar x.add_(2) into x.conj_().add_(2)`?

Reply:
flipping the bit is required to ensure other tensors dependent on it resolve to the correct value